test/with_api_v2/acceptance.go

```diff
@@ -384,7 +384,7 @@ func (amc *AlertmanagerCluster) Terminate() {
 func (am *Alertmanager) Terminate() {
 	am.t.Helper()
 	if err := syscall.Kill(am.cmd.Process.Pid, syscall.SIGTERM); err != nil {
-		am.t.Fatalf("Error sending SIGTERM to Alertmanager process: %v", err)
+		am.t.Logf("Error sending SIGTERM to Alertmanager process: %v", err)
```

Comment:
There's no reason why this should panic -- in either case, there's no action for us to take.